### ISSUE TITLE
[BUG]: Module Name Truncated in Logs

### ISSUE THREAD

**Issue by Stillhard** [PerformanC/ReZygisk#69]
### Version

v1.0.0-270-37c40f4-release

### Modules

magisk_proc_monitor by [HuskyDG](https://github.com/Magisk-Modules-Alt-Repo/magisk_proc_monitor/)

### Description

module name truncated

### Steps to reproduce

See logs

### Logs

```
2024-11-04 09:58:23.841   782-782   zygiskd64               zygiskd64                            I    Polling companion for module "magisk_proc_monitor"
2024-11-04 09:58:23.842   782-782   zygiskd64               zygiskd64                            I    Sending companion fd socket of module "magisk_proc_monitor"
2024-11-04 09:58:23.842   782-782   zygiskd64               zygiskd64                            I  ZD++ RequestCompanionSocket
2024-11-04 09:58:23.842  3046-3046  zygiskd64               zygiskd64-magisk_proc_monitor        I  New companion request.
                                                                                                     - Module name: m
                                                                                                     - Client fd: 8
```
HERE -> Module name: m
shouldn't it be ->  Module name: magisk_proc_monitor ?

### Confirmations

- [X] My environment meets the minimum requirements.
- [X] I have verified that this is not a duplicate issue.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ThePedroo** [comment]
Had found before but still haven't pushed to the GitHub repository yet

**Comment by ThePedroo** [comment]
Fixed on 3265bcb5812a0801e77f01c21884793ee73ee2c3.
